docs/axes/README.md

```diff
@@ -31,7 +31,7 @@ There are a number of config callbacks that can be used to change parameters in
 | `beforeDataLimits` | `axis` | Callback that runs before data limits are determined.
 | `afterDataLimits` | `axis` | Callback that runs after data limits are determined.
 | `beforeBuildTicks` | `axis` | Callback that runs before ticks are created.
-| `afterBuildTicks` | `axis` | Callback that runs after ticks are created. Useful for filtering ticks.
+| `afterBuildTicks` | `axis`, `ticks` | Callback that runs after ticks are created. Useful for filtering ticks (*Should return the filtered ticks*).
```

Comment:
I'm wondering if this really needs to be in bold? It also seems to me like the parentheses aren't needed. Like the bold is saying it's really important, but then the parentheses are saying it's just a side note. Seems almost like two contradictory formattings. Also, I think if you were to keep it in bold that github uses two stars instead of one to denote bold text

I.e. maybe `Useful for filtering ticks. Should return the filtered ticks.` instead of `Useful for filtering ticks (*Should return the filtered ticks*).`?